usb/esp_tinyusb/Kconfig

```diff
@@ -241,21 +241,11 @@ menu "TinyUSB Stack"
                 BTH ISO ALT COUNT.
     endmenu # "Bluetooth Host Device Class"
 
-    menu "Network driver (ECM/NCM/RNDIS)"
-        choice TINYUSB_NET_MODE
-            prompt "Network mode"
-            default TINYUSB_NET_MODE_NONE
+    menu "Network driver (NCM)"
+        config TINYUSB_NET_MODE_NCM
+            bool "Enable TinyUSB Network feature(NCM)"
+            default n
             help
-                Select network driver you want to use.
-
-            config TINYUSB_NET_MODE_ECM_RNDIS
```

Comment:
Could you please explain why do we remove the option to use ECM/RNDIS?

I think we can leave the option there, even though we don't support it in `esp_tinyusb`. This way, the user can still implement it himself, in case he does not want to use NCM

Reply:
NCM can be used on linux/MAC/windows, ECM/RNDIS do duplicate work. In addition, if ECM/RNIDS need to be supported on both windows(Only support RNDIS) and MAC(Only support ECM),  we need to use two descriptor configurations, but esp_tinyusb can only support one configurations.
I removed this option because I thought it would be weird for customers to think we had already implemented it